app/controllers/teammates_controller.rb

```diff
@@ -0,0 +1,60 @@
+class TeammatesController < ApplicationController
+  before_action :require_invitation
+  before_action :require_pending
+  helper_method :already_teammate
+  rescue_from ActiveRecord::RecordNotFound, :with => :incorrect_token
+
+  def accept
+    if current_user && !already_teammate?
```

Comment:
you have two checks for already_teammate. Is that necessary?  Can you reduce that to one?